shortnames/shortnames.conf

```diff
@@ -18,13 +18,8 @@
   "gpt-oss:20b" = "hf://ggml-org/gpt-oss-20b-GGUF"
   "gpt-oss:120b" = "hf://ggml-org/gpt-oss-120b-GGUF"
   "granite" = "ollama://granite3.1-dense"
-  "granite-code" = "hf://ibm-granite/granite-3b-code-base-2k-GGUF/granite-3b-code-base.Q4_K_M.gguf"
```

Comment:
Why remove these and fall back to trusting Ollama?

Reply:
Previously the chat templates were broken for the ollama models.

Eric tried to workaround in https://github.com/containers/ramalama/commit/da7eb540466b87b7688e22c88ed7feaa7b639a22 but chose the base models instead of the instruct models. The chat API doesn't work with the base models.

Can either revert back to the ollama models (which now work) or chose instruct models from HF.

Reply:
ok